#request
---
#response
uint16 count
float64[] x
float64[] y
float64[] z